README.md

```diff
@@ -14,14 +14,10 @@ A **development container** is a running [Docker](https://www.docker.com) contai
 This repository contains a set of **dev container templates** which are source files packaged together that encode configuration for a complete development environment. A template can be used in a new or existing project, and a [supporting tool](https://containers.dev/supporting) will use the configuration from the template to build a development container.
 
 ⚠️ Development container 'templates' are a
-[**proposed**](https://github.com/devcontainers/spec/blob/main/proposals/devcontainer-templates.md) addition to the
-[development container specification](https://containers.dev/implementors/spec/). **Please note that templates are in
+[**proposed**](https://containers.dev/implementors/templates/) addition to the
+[Dev Container Template specification](https://containers.dev/implementors/spec/). **Please note that templates are in
```

Comment:
```suggestion
[Dev Container Template specification](https://containers.dev/implementors/spec/). **Please note that Templates are in
```

Reply:
+1 to what @Chuxel mentions though - perhaps we just remove this section.

Reply:
Just Removed!